### ISSUE TITLE
[Bug] EmptyTocItemName Warnings When using Xrefs 

### ISSUE THREAD

**Issue by groogiam** [dotnet/docfx#9649]
**Describe the bug**
As of 2.75.1+6aa697f56975e85e82c5a6b81b71157c77302270 I am getting warnings when I use uid's in my toc.

**To Reproduce**

toc.md
```
# <xref:custom-entity-add-new>

```

page.md

```
---
uid: custom-entity-add-new
---

# Add New Domain Entity

My Content
```
This results in the following warning

`warning: EmptyTocItemName: TOC item (TopicUid:custom-entity-add-new ) with empty name found. Missing a name?`

**Expected behavior**
The referenced file has a header # so I would expect this warning to not be shown as was done with previous versions.

**Context (please complete the following information):**
- OS: Windows
- Docfx version: 2.75.1+6aa697f56975e85e82c5a6b81b71157c77302270

**Comment by filzrev** [comment]
This issue seems be fixed by 9665.

**Comment by groogiam** [comment]
@filzrev Thanks for the update. Is this released in [2.75.3](https://www.nuget.org/packages/docfx/2.75.3)? If not any idea when the next release will be? Thanks.

**Comment by filzrev** [comment]
It seems be included in v2.75.3.
Please let me know if there are any problems.

**Comment by groogiam** [comment]
Everything appears to be working. Thanks for your help!
